data/names.txt

```diff
@@ -2158,6 +2159,7 @@ phrase "pirate"
 		"Bony Fingers"
 		"Yo Ho Ho"
 		"Low Blow"
+		"bilgewater"
```

Comment:
lacks a capitalized letter, should be "Bilgewater"

Reply:
Already fixed in the latest commit!

Reply:
oops, my bad :P